go.mod

```
@@ -19,16 +23,16 @@ require (
 	github.com/fluxcd/notification-controller/api v1.5.0
 	github.com/fluxcd/pkg/apis/event v0.17.0
 	github.com/fluxcd/pkg/apis/meta v1.12.0
-	github.com/fluxcd/pkg/auth v0.14.0
-	github.com/fluxcd/pkg/chartutil v1.3.0
+	github.com/fluxcd/pkg/auth v0.16.0
+	github.com/fluxcd/pkg/chartutil v1.4.0
```

```suggestion
	github.com/fluxcd/pkg/chartutil v1.3.0
```

To avoid bumping Helm.

---

Maybe better to downgrade to keep everything at pkg latest? https://github.com/fluxcd/pkg/pull/933

---

Fixed, thanks!